README.md

```diff
@@ -64,17 +64,11 @@ Key:
 
 ## Guides
 
-### User Guide
-
 See our documentation on [kubeedge.io](https://kubeedge.io).
 
 **Quick Start** - Install KubeEdge with [keadm](./docs/setup/keadm.md).
 
-Try some examples of KubeEdge on [examples](https://github.com/kubeedge/examples).
-
-### Developer Guide
```

Comment:
Please leave the `Developer Guide` and `User Guide` here, for more clearly distinguish users.

Reply:
Developer Guide could be duplicate with `contribute.md` i think.